### DISCUSSION TITLE
Update client from patchman

### DISCUSSION THREAD

**Discussion by u-capg** [furlongm/patchman#555] [General]
Hi
I have installed patchman on server 
I would to update my server from patchman web console but i have some error when i try 
I go on package update, select the package to update with old version and new version but
i have error 

it's possible t update server from web console?

Giovanni

**Comment by u-capg** [comment]
i could integrate with puppet?

**Comment by furlongm** [comment]
patchman doesn't do the updating on your clients, it just lets you know what updates are available
